### DISCUSSION TITLE
Notification about structure change

### DISCUSSION THREAD

**Discussion by ah-rahimi** [def-studio/telegraph#560] [Ideas]
Please, when changing the config file, tell us to change it in the change log
After searching for bugs, I found out that the structure of the config file has changed

Thanks 🙏

**Comment by fabio-ivona** [comment]
Hi, the change should have been backward compatible using fallback on old structure

which key was not handled?

**Reply by ah-rahimi** [reply]
Hi,
Yes, right. I checked the code and saw that the previous structure is supported.
But it was strange that it could not use my previous webhook_handler and the bot did not work perfectly because it was using empty webhook handle.

**Reply by fabio-ivona** [reply]
hmm, really weird, maybe I've missed something, will check it and fix any backward compatibility issues on webhooks, thanks!
